string-manipulation-3/src/is-anagram.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-unused-vars */
+function isAnagram(firstString, secondString) {
+  var newFirst = firstString.replaceAll(' ', '');
+  var newSecond = secondString.replaceAll(' ', '');
+
+  if (newFirst.length !== newSecond.length) return false;
+
+  for (var i = 0; i < newFirst.length; i++) {
+    var statement = newFirst.includes(newSecond[i]);
+    if (statement === false) return statement;
+  }
+  return statement;
```

Comment:
`includes()` is a little misleading here, because it stops at the _first_ occurrence of a character. So if you check for two `'j'`s, they both return `true` even if the other string only has one.

If you split, sort, and re-join the characters in each string, you can just compare them to each other.